packages/datadog-instrumentations/src/helpers/hooks.js

```diff
@@ -1,135 +1,144 @@
+    'jest-worker': () => require('../jest'),
+    knex: () => require('../knex'),
+    koa: () => require('../koa'),
+    'koa-router': () => require('../koa'),
+    kafkajs: () => require('../kafkajs'),
+    langchain: () => require('../langchain'),
+    ldapjs: () => require('../ldapjs'),
+    'limitd-client': () => require('../limitd-client'),
+    lodash: () => require('../lodash'),
+    mariadb: () => require('../mariadb'),
+    memcached: () => require('../memcached'),
+    'microgateway-core': () => require('../microgateway-core'),
+    mocha: () => require('../mocha'),
+    'mocha-each': () => require('../mocha'),
+    moleculer: () => require('../moleculer'),
+    mongodb: () => require('../mongodb'),
+    'mongodb-core': () => require('../mongodb-core'),
+    mongoose: () => require('../mongoose'),
+    mquery: () => require('../mquery'),
+    multer: () => require('../multer'),
+    mysql: () => require('../mysql'),
+    mysql2: () => require('../mysql2'),
+    net: () => require('../net'),
+    next: () => require('../next'),
+    'node-serialize': () => require('../node-serialize'),
+    'node:child_process': () => require('../child_process'),
+    'node:crypto': () => require('../crypto'),
+    'node:dns': () => require('../dns'),
+    'node:http': () => require('../http'),
+    'node:http2': () => require('../http2'),
+    'node:https': () => require('../http'),
+    'node:net': () => require('../net'),
+    'node:url': () => require('../url'),
+    'node:vm': () => require('../vm'),
+    nyc: () => require('../nyc'),
+    oracledb: () => require('../oracledb'),
+    openai: { esmFirst: true, fn: () => require('../openai') },
+    paperplane: () => require('../paperplane'),
+    passport: () => require('../passport'),
+    'passport-http': () => require('../passport-http'),
+    'passport-local': () => require('../passport-local'),
+    pg: () => require('../pg'),
+    pino: () => require('../pino'),
+    'pino-pretty': () => require('../pino'),
+    playwright: () => require('../playwright'),
+    'playwright-core': () => require('../playwright'),
+    'promise-js': () => require('../promise-js'),
+    promise: () => require('../promise'),
+    protobufjs: () => require('../protobufjs'),
+    pug: () => require('../pug'),
+    q: () => require('../q'),
+    redis: () => require('../redis'),
+    restify: () => require('../restify'),
+    rhea: () => require('../rhea'),
+    router: () => require('../router'),
+    'selenium-webdriver': () => require('../selenium'),
+    sequelize: () => require('../sequelize'),
+    sharedb: () => require('../sharedb'),
+    tedious: () => require('../tedious'),
+    undici: () => require('../undici'),
+    url: () => require('../url'),
+    vitest: { esmFirst: true, fn: () => require('../vitest') },
+    vm: () => require('../vm'),
+    when: () => require('../when'),
+    winston: () => require('../winston'),
+    workerpool: () => require('../mocha')
+  },
+  DISABLED_BY_DEFAULT_INTEGRATIONS
```

Comment:
I don't think we'd need to make this object property all caps by default since it's no more of a constant than the `.hooks` object property:

```suggestion
  disabledByDefaultIntegrations: DISABLED_BY_DEFAULT_INTEGRATIONS
```

Reply:
Why not just use a property instead similar to `serverless` and `esmFirst`?